### PR TITLE
Update README.md - Fix vcpkg command

### PR DESCRIPTION
**Pull request by porkedpie**
Missing word "install" in command to get vcpkg to install the dependences.

# Issue

Windows installers following the readme will not be able to get vcpkg to work without the word install in the command

## Tasklist

Just the readme

## Requirements / Relations

None - readme
 Link any requirements here. Other pull requests this PR is based on?

### CODE REVIEW

**Comment by porkedpie** [comment]
Also just added the build command for Windows users.

**Comment by kevinkreiser** [comment]
this looks good to me but ill let others who more routinely build on windows have a look

**Comment by nilsnolde** [comment]
oops, thanks for fixing and extending!

**Review by nilsnolde — APPROVED** [review]

**Review by mloskot — APPROVED** [review]

**Comment by nilsnolde** [comment]
while you're at it (and before kevin continues on the merging spree;)), can you please just quickly add this line to the README  @porkedpie :

```
cd /path/to/vcpkg
git checkout f4bd6423
```

right before `C:\path\to\vcpkg.exe --triplet x64-windows install "@.vcpkg_deps.txt"` would be best. thanks!

**Comment by nilsnolde** [comment]
`git checkout f4bd6423` is the important line, you accidentally removed that @porkedpie

**Comment by porkedpie** [comment]
Sorry, was making a bit of a mess of it

**Review by nilsnolde — DISMISSED** [review]

**Comment by nilsnolde** [comment]
happens.. just sad for the CI hours. ideally they wouldn't even trigger on source unrelated changes..

given how often that's happening, it might be worth to include a small script to skip CI builds? e.g. https://reflectoring.io/skip-ci-build/#using-a-git-diff-in-the-ci-build and for circle CI to get the commit range https://discuss.circleci.com/t/get-list-of-commits-in-build/15725

File: README.md
Comment (nilsnolde) [inline README.md:169]: You'll have to reference the full path here for `vcpkg.exe`, otherwise you assume its parent dir is in `PATH` (which it's mostly not)

**Review by nilsnolde — CHANGES_REQUESTED** [review]
Sorry, lost track on this. Good to go, just that one small nitty modification. No need for changelog entry IMHO

File: README.md
Comment (mloskot) [inline README.md:178]: I'd rather present CMake 3.13+ way:

```
cmake -S  C:\path\to\project -B  C:\path\to\build
```
or
```
cd C:\path\to\project 
cmake -B build .
```

**Review by mloskot — APPROVED** [review]

**Comment by kevinkreiser** [comment]
@nilsnolde or @mloskot would either of you be able to make the requested change so we could get this merged in?

**Comment by nilsnolde** [comment]
since I'm at it I'll do it

File: README.md
Comment (nilsnolde) [reply README.md]: done

File: README.md
Comment (nilsnolde) [reply README.md]: done

**Comment by kevinkreiser** [comment]
no need to wait for the build on this one lets just merge it in!

**Comment by mloskot** [comment]
Sorry folks, I'm a bit off the line with everything.

**Pull request merged**
